COPYING

```diff
@@ -1,4 +1,4 @@
-# License 1
+# License 1 (The 3-Clause BSD License)
```

Comment:
```suggestion
# License 1

The 3-Clause BSD License
```

I see this is how it's written by default when asking github to do it. Maybe someone else is looking for it on its own line. I suppose we should also update License 2 in this file to say the same thing?

Reply:
I think adding BSD 3-Clause License words in License 2 part is fine as well.  I think there is no other specific words. Thanks